rl_chain/__init__.py

```diff
@@ -1,10 +1,20 @@
-from . import pick_best_chain
-from . import slates_chain
+from .pick_best_chain import (
+    PickBest,
+    PickBestAutoSelectionScorer,
+    PickBestFeatureEmbedder,
```

Comment:
maybe we should not expose embedders from top level?
- inheritance from them seems quite advanced scenario, so they may confuse
- not clear why inheritance from them may be needed since they have almost no state and only one function

Reply:
yeah I can remove